client/src/components/DataFiles/DataFilesModals/DataFilesMkdirModal.js

```diff
@@ -8,11 +8,14 @@ import { Button, Modal, ModalHeader, ModalBody, ModalFooter } from 'reactstrap';
 
 const DataFilesMkdirModal = () => {
   const dispatch = useDispatch();
+  const systemList = useSelector(state => state.systems.systemList);
+  const isOpen = useSelector(state => state.files.modals.mkdir);
   const params = useSelector(
     state => state.files.params.FilesListing,
     shallowEqual
   );
-  const isOpen = useSelector(state => state.files.modals.mkdir);
+  const currentSystem =
```

Comment:
We have a utility function for this in `utils/systems` called `findSystemDisplayName` that is currently used in other places where system name is needed.  That would work for the current state of master (where there are no projects yet)

--------------
But as @jchuahtacc  pointed out in his comment, projects systems won't appear in our systemList and `findSystemDisplayName` would return just the host name due to that.

We need to extend things so that look both at **the list of projects** and **the systems list** and then determine if we have a display name that we can translate?

should something like this be done in this PR/FP-664?
or should we have a separate follow-on-issue issue that we tackle in Projects epic.  @jchuahtacc,  @jarosenb, @rstijerina, @Strmiska

Reply:
Thanks for catching this @nathanfranklin and @jchuahtacc , I'm not sure the best way to handle this without projects being in place to test. I would think we should make something like `findSystemDisplayName` that could also account for some other shared workspaces/project, since we'd probably have to replace this in multiple areas. Or would we only need this for file management and job outputs?

Reply:
@Strmiska Can you test this now with projects in place?